modules/about-dynamic-plug-ins.adoc

```diff
@@ -0,0 +1,34 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/web_console/dynamic-plug-ins.adoc
+
+:_content-type: CONCEPT
+[id="about-dynamic-plugins_{context}"]
+= About dynamic plug-ins
+
+A dynamic plug-in allows you to add custom pages and other extensions to your interface at runtime. Plug-ins are registered with the console using the `ConsolePlugin` custom resource, and they are enabled in the console operator configuration by a cluster administrator.
+
+[id="dynamic-plugins-features"]
+== Key features
+
+A dynamic plug-in will allow you to make customizations such as:
+
+* Add custom pages
+* Add perspectives and update navigation items
+* Add tabs and actions to resource pages
+
+== PatternFly 4 Guidelines
+When creating your plug-in, follow these guidelines for using PatternFly:
+
+* Use link:https://www.patternfly.org/v4/[PatternFly4] components and PatternFly CSS variables. Core PatternFly components are available through the SDK. Using PatternFly components and variables will help your plug-in look consistent in future console versions.
+* Make your plug-in accessible by following link:https://www.patternfly.org/v4/accessibility/accessibility-fundamentals/[PatternFly's accessibility fundamentals].
+* _Do not_ use other CCS libraries such as Bootstrap or Tailwind. They can conflict with PatternFly and will not match the console look and feel.
+
+[id="general-plug-in-guidelines"]
+== General Guidelines
+When creating your plug-in, follow these general guidelines:
+
+* Prefix your CSS class names with your plug-in name to avoid collisions. For example, `my-plugin__heading` and `my-plugin__icon`.
```

Comment:
I'm not sure the right asciidoc syntax to escape the `__`. It's still showing up as italics. I think perhaps `\`? (untested)

```suggestion
* Prefix your CSS class names with your plug-in name to avoid collisions. For example, `my-plugin\_\_heading` and `my-plugin\_\_icon`.
```

Reply:
Looks like that worked. I will have some one on our peer review rotation check it. Thanks!